gst/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -40,6 +67,71 @@ struct _Tflite_data
 };
 typedef struct _Tflite_data tflite_data;
 
+/**
+  * @brief parse custom options for tensorflow lite
+  * @param filter : tensor_filter instance
+  * @param id : Option ID
+  * @return 0 or 1 if successfully parsed.
+  *        -1 if parsing a value of a given option failed.
+  */
+static int
+tflite_get_parser_boolean (const GstTensorFilter * filter, int id)
+{
+  int ret = -1;
+
+  if (filter->prop.custom_properties == NULL)
+    return ret;
+
+  if (id < 0 || id >= TFLITE_OPTION_MAX) {
+    GST_WARNING ("Invalid option id. (%d)\n", id);
+    return ret;
+  }
+
+  custom_options = g_strdup (filter->prop.custom_properties);
```

Comment:
If filter->prop.custom_properties is NULL, g_strdup returns NULL so that custom_options is also NULL. IMO, since you've already checked whether filter->prop.custom_properties is NULL or valid at line 82, this code looks unnecessary.